### PR TITLE
Update README.md for install issue

### PR DESCRIPTION
**Pull request by kyuksel**
I ran into an error while setting up my environment and found the solution in a response to a [Slack message](https://broadinstitute.slack.com/archives/CA3HY2Q3S/p1589999388163400), and thought it'd be good to add that info directly to the instructions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,6 +39,26 @@ Requests related to the funtionality or usability of the UI can be submitted as
...
+
+    ./node_modules/react-dev-utils/webpackHotDevClient.js
+    Error: [BABEL] /Users/.../terra-ui/node_modules/react-scripts/node_modules/react-dev-utils/webpackHotDevClient.js: Cannot find module '...'
+    Require stack:
+    - ...
+    - ...
+    - ... (While processing: "...js")
+    ```
+    
+    try
```
Comment (zarsky-broad) [inline README.md]:
```suggestion
    If you get an error that resembles:
    
    ```
    Failed to compile.

    ./node_modules/react-dev-utils/webpackHotDevClient.js
    Error: [BABEL] /Users/.../terra-ui/node_modules/react-scripts/node_modules/react-dev-utils/webpackHotDevClient.js: Cannot find module '...'
    Require stack:
    - ...
    - ...
    - ... (While processing: "...js")
    ```
    
    try:

```

**Review by zarsky-broad — APPROVED** [review]
One suggestion for tidying up

**Review by ehanna4 — APPROVED** [review]
Other than the one change suggestion this looks good to go. Thanks for catching this and tracking down the fix and adding it!

**Comment by kyuksel** [comment]
Thanks for the reviews @zarsky-broad & @ehanna4

**Pull request merged**